dev-core

```diff
@@ -4,8 +4,10 @@ set -e
 
 DIR=$(pwd)
 MESG_LOG_LEVEL=${MESG_LOG_LEVEL:-debug}
+MESG_LOG_FORCECOLORS=${MESG_LOG_FORCECOLORS:-true}
```

Comment:
Set default to false otherwise we have colors by default and we need explicitly to set MESG_LOG_FORCECOLORS=false to have the json version

```suggestion
MESG_LOG_FORCECOLORS=${MESG_LOG_FORCECOLORS:-false}
```

Reply:
Ok, I thought we want to have color by default ;)

Reply:
no only when we put the `FORCECOLORS` env variable. Basically we just add a new type of log